### PR TITLE
Remove jQuery class from the comment context menu

### PR DESCRIPTION
**Pull request by yardenshoham**
- Switched from jQuery class functions to plain JavaScript
- Tested the comment context menu functionality and it works as before

### CODE REVIEW

**Review by silverwind — APPROVED** [review]
Next time, batch a few more of these together, they are trivial changes.

**Comment by yardenshoham** [comment]
I'm doing these file by file to make reviewing and testing easier

File: web_src/js/modules/fomantic/dropdown.js [hunk 1]
```diff
@@ -207,7 +207,7 @@ function attachDomEvents(dropdown, focusable, menu) {
       if (!$item) $item = $(menu).find('> .item.selected'); // when dropdown filters items by input, there is no "value", so query the "selected" item
       // if the selected item is clickable, then trigger the click event.
       // we can not click any item without check, because Fomantic code might also handle the Enter event. that would result in double click.
-      if ($item && ($item[0].matches('a') || $item.hasClass('js-aria-clickable'))) $item[0].click();
+      if ($item && ($item[0].matches('a') || $item[0].classList.contains('js-aria-clickable'))) $item[0].click();
```
Comment (silverwind) [inline web_src/js/modules/fomantic/dropdown.js]:
```suggestion
      if ($item?.[0]?.matches('a.js-aria-clickable')) $item[0].click();
```

**Comment by silverwind** [comment]
> I'm doing these file by file to make reviewing and testing easier

Right, keep at it then. Makes it easier to deeper review the small changesets.

File: web_src/js/modules/fomantic/dropdown.js (hunk 1 above)
Comment (yardenshoham) [reply web_src/js/modules/fomantic/dropdown.js]: Broken now

Comment (yardenshoham) [reply web_src/js/modules/fomantic/dropdown.js]: The check is an `OR` between `a` and `js-aria-clickable`, not an `AND`

Comment (yardenshoham) [reply web_src/js/modules/fomantic/dropdown.js]: Fixed https://github.com/go-gitea/gitea/pull/30179/commits/07c3d1eff6efcd1f381754a49d682ad29aaa6bf0

**Review by silverwind — APPROVED** [review]

File: web_src/js/modules/fomantic/dropdown.js [hunk 2]
```diff
@@ -207,7 +207,7 @@ function attachDomEvents(dropdown, focusable, menu) {
       if (!$item) $item = $(menu).find('> .item.selected'); // when dropdown filters items by input, there is no "value", so query the "selected" item
       // if the selected item is clickable, then trigger the click event.
       // we can not click any item without check, because Fomantic code might also handle the Enter event. that would result in double click.
-      if ($item && ($item[0].matches('a') || $item.hasClass('js-aria-clickable'))) $item[0].click();
+      if ($item?.[0]?.matches('a, .js-aria-clickable')) $item[0].click();
```
Comment (silverwind) [inline web_src/js/modules/fomantic/dropdown.js:210]: Why `, .`? Are we not looking for a `a.js-aria-clickable`?

Comment (yardenshoham) [reply web_src/js/modules/fomantic/dropdown.js:210]: The original code doesn't

Comment (silverwind) [reply web_src/js/modules/fomantic/dropdown.js:210]: Oh I see, I missed that this is `||`, not `&&`.

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**